### ISSUE TITLE
Angular 1.7.7 doesn't load in Safari < 5.0.6

### ISSUE THREAD

**Issue by kabilanvk** [angular/angular.js#16874]
Angular 1.7.7 throws "$$SanitizeUriProvider is not a constructor" error in older safari browsers.

# AngularJS is in LTS mode
We are no longer accepting changes that are not critical bug fixes into this project.
See https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c for more detail.

**I'm submitting a ...**

- [x] regression from 1.7.0
- [ ] security issue
- [ ] issue caused by a new browser version
- [ ] other 

**Current behavior:**
Exception thrown from older safari browsers (Used as embedded browser component from standalone installer)

**Expected / new behavior:**
Angular should load with out any issues.

**Minimal reproduction of the problem with instructions:**
Will try to create one.

**AngularJS version:** 1.7.7

**Browser:** [Safari <5.0.6 ]
Mac OS X 10.5 Leopard | 5.0.6

**Anything else:**
[$injector:modulerr] Failed to instantiate module ng due to:
'function $$SanitizeUriProvider() {
    [native code]
}' is not a constructor
https://errors.angularjs.org/1.7.7/$injector/modulerr?p0=ng&p1='function%20%24%24SanitizeUriProvider()%20%7B%0A%20%20%20%20%5Bnative%20code%5D%0A%7D'%20is%20not%20a%20constructor

**Comment by Narretz** [comment]
Thanks for the report. However, we are not going to accept this bug:
For evergreen (auto-updating) browsers, we officially support only the 2 latest versions, and Safari is part of those (even though the release is yearly, and not 6 weeks like Chrome and Firefox). So at the point of the change that broke Safari 5.0.6 (AngularJS 1.7.0), we only supported Safari 11 and 10. (Moreover, Safari 5.0.6 was released a full year before AngularJS 1.0)
